.github/workflows/e2e-init-chain.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Push E2E Init Chain Images
+
+on:
+  push:
+    tags:
+    - 'v12+.[0-20]+.0+' 
+    - 'v[0-20]+.0+.0+' 
+
+env:
+  DOCKER_REPOSITORY: osmolabs/osmosis-e2e-init-chain
```

Comment:
Consider naming the file `docker-e2e.yml` please

This is to make it more intuitive about the similarity to `docker.yml` while still being clear that this is for e2e

Reply:
OK, I'll fix it right now